data/transition-sites/dclg_localdirect.yml

```diff
@@ -0,0 +1,9 @@
+---
+site: dclg_localdirect
+whitehall_slug: department-for-communities-and-local-government
+homepage: https://www.gov.uk/government/organisations/department-for-communities-and-local-government
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
```

Comment:
The site [does have some TNA crawls](http://webarchive.nationalarchives.gov.uk/*/http://www.localdirect.gov.uk), so we can add a real timestamp here.

According to the [ticket](https://govuk.zendesk.com/agent/tickets/1249417) a final TNA crawl is under way and we'll need to update the timestamp again before the site transitions, so could you add a comment to that effect here?